racetrack_commons/racetrack_commons/api/metrics.py

```diff
@@ -1,13 +1,35 @@
+from a2wsgi import WSGIMiddleware
 from fastapi import FastAPI
-from prometheus_client.exposition import make_asgi_app
+from prometheus_client import Counter, Histogram
+from prometheus_client.exposition import make_wsgi_app
 from prometheus_client.registry import REGISTRY
+
 from racetrack_commons.api.asgi.proxy import TrailingSlashForwarder
 
+metric_internal_server_errors = Counter(
+    'commons_internal_server_errors',
+    'Number of Internal Server Errors (500) caught by error handler',
+)
+metric_request_duration = Histogram(
+    'commons_request_duration',
+    'Duration of API requests',
```

Comment:
Nitpick: how about adding "(in seconds)". I'm not sure how obvious it is in the actual dashboard what is the unit of time, so feel free to ignore this.

Reply:
Of course! It's a completely valid remark. It's a MUST.